assets/respec2.css

```diff
@@ -134,6 +134,12 @@ table.simple {
   text-align: left;
 }
 
+.simple th a:link {
```

Comment:
Should we remove `:link`? Because otherwise clicking links will restore them as blue.

Reply:
Ah true. I should check :visited too I guess.